baselines/monitoring/src/v3/metric_service_client.ts.baseline

```diff
@@ -150,7 +150,7 @@ export class MetricServiceClient {
       folderGroupPathTemplate: new this._gaxModule.PathTemplate(
         'folders/{folder}/groups/{group}'
       ),
-      folderMetricDescriptorPathTemplate: new this._gaxModule.PathTemplate(
+      folderMetricDescriptorsMetricDescriptorPathTemplate: new this._gaxModule.PathTemplate(
```

Comment:
This looks wrong :) Maybe the `=**` part is confusing the logic?

Reply:
yes, match pattern does not include this case. Now it's changed back :)